person.rb

```diff
@@ -17,4 +20,8 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def validate_name(name)
+    return @corrector.correct_name(name) unless @corrector
+  end
```

Comment:
- [ ] You don't need to pass a parameter when defining this method. The purpose of this method in the **Person** class is to correct the `@name` instance variable (note the `@`). Instead of this, you should have something similar to this:

  ```
  def validate_name
    @name = @corrector.correct_name(@name)
  end
  ```

Reply:
Thanks, I had trouble here.